examples/with-typescript-graphql/lib/graphql-operations.ts

```diff
@@ -0,0 +1,54 @@
+import { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/core';
```

Comment:
Shouldn't generated files be ignored from version control?

Reply:
You're right, fixed in [f9ce8cb](https://github.com/vercel/next.js/pull/36240/commits/f9ce8cb28d05e5598204af98d60e4517a63d2ab7)